clean.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
```

Comment:
What is this file for ? don't think we need to add extra files.

Reply:
it is for my JetBrains environment to check shell cmds in files via shellcheck